lib/active_merchant/billing/gateways/sum_up.rb

```diff
@@ -157,7 +159,9 @@ def parse(body)
       end
 
       def success_from(response)
-        return false unless %w(PENDING EXPIRED PAID).include?(response[:status])
+        return false unless %w(PENDING EXPIRED PAID REFUND).include?(response[:status])
+
+        return true if response[:transactions].blank?
```

Comment:
since you are returning a proper response for `refund`, I think this `return true `is not needed, unless for other transaction we have empty responses.